src/Composer/TravisDrupalModulePlugin.php

```diff
@@ -78,25 +78,41 @@ final class TravisDrupalModulePlugin implements
      * Apply plugin modifications to Composer
      *
      * @param Composer    $composer The composer API.
-     * @param IOInterface $theIo    IO interface to log to the terminal.
+     * @param IOInterface $io       IO interface to log to the terminal.
```

Comment:
This used to fail PHP EA validatio because it was too short. Glad to see that nonsense gone.

Reply:
It's not gone, I forced the commit even with it 😊